java/java.hints/nbproject/project.xml

```diff
@@ -165,6 +165,14 @@
                         <specification-version>1.28</specification-version>
                     </run-dependency>
                 </dependency>
+                <dependency>
+                    <code-name-base>org.netbeans.modules.editor.tools.storage</code-name-base>
+                    <build-prerequisite/>
+                    <compile-dependency/>
+                    <run-dependency>
+                        <specification-version>1.31</specification-version>
+                    </run-dependency>
+                </dependency>
```

Comment:
this looks like the cause for the build failure:
```
BUILD FAILED
/home/runner/work/netbeans/netbeans/java/java.lsp.server/build.xml:32: The following error occurred while executing this line:
/home/runner/work/netbeans/netbeans/nbbuild/netbeans/harness/suite.xml:163: The following error occurred while executing this line:
/home/runner/work/netbeans/netbeans/nbbuild/netbeans/harness/build.xml:150: Module org.netbeans.modules.editor.tools.storage excluded from the target platform; the path is: [org.netbeans.modules.nbcode.integration, org.netbeans.modules.java.lsp.server, org.netbeans.modules.java.hints]
```

Reply:
So, this is should be removed from diasabled modules list of vscode extension?

Reply:
I don't know. cc @sdedic @dbalek

Reply:
I tried removing `org.netbeans.modules.editor.tools.storage` from disabled modules list and it is working fine. I can update the branch and push the code if this PR is reverted.